scanpy/plotting/_baseplot_class.py

```diff
@@ -347,7 +347,7 @@ def add_totals(
         >>> adata = sc.datasets.pbmc68k_reduced()
         >>> markers = {'T-cell': 'CD3D', 'B-cell': 'CD79A', 'myeloid': 'CST3'}
         >>> plot = sc.pl._baseplot_class.BasePlot(adata, markers, groupby='bulk_labels').add_totals()
-        >>> plot.plot_group_extra['counts_df']
+        >>> plot.plot_group_extra['counts_df']  # doctest: +SKIP
```

Comment:
Not a huge fan of outright skipping this. I understand why you do it (changed Series repr), but I don‘t want this to get out of sync with the code before it.

Reply:
I spent some time looking to see if we could optionally skip based on the test job or pandas version, but it didn't seem very straightforward.

Reply:
I would also like to not skip it

Reply:
I think a custom doctest directive that skips the line based on the pandas version would be pretty ideal here.

docktest-plus sorta has something like this: https://github.com/scientific-python/pytest-doctestplus?tab=readme-ov-file#doctest-dependencies

But the inline version doesn't work in function docstrings, only `.rst` files AFAICT.

Reply:
OK, I just commented in the doctest-plus issue: https://github.com/scverse/scanpy/issues/2729#issuecomment-1934479845